### ISSUE TITLE
Moravian Lives schema docs - fix NERVE behaviour

### ISSUE THREAD

**Issue by ilovan** [cwrc/CWRC-WriterBase#233]
Document: https://dev-cwrc-writer.cwrc.ca/?githubPath=TestGitWriterSchemaCSSchange&githubRepo=ilovan%2FGit-Writer-tests

Two Errors:

1. NERVE identifies everything as person:
![image.png](https://images.zenhubusercontent.com/5a5e621f4b5806bc2bc2ff7d/9327f2a8-7cb1-4f7c-b0b9-9e4cffa8ee52)

2. Once the NERVE-generated entities are all edited and accepted, they are all saved as RS tags. 
![image.png](https://images.zenhubusercontent.com/5a5e621f4b5806bc2bc2ff7d/5bfd2196-36d2-4edb-95c0-0a3e0aacc55f)

Moravian Lives has the same entity mappings as the schemas listed here: https://github.com/cwrc/CWRC-WriterBase/issues/120
Moravian Lives itself was set up by @lucaju : https://github.com/cwrc/CWRC-GitWriter/issues/65

May be related to how the handling is set up at: https://github.com/cwrc/CWRC-WriterBase/issues/194

**Comment by ajmacdonald** [comment]
Duplicate of https://github.com/cwrc/CWRC-WriterBase/issues/230 which is fixed, but not yet deployed

**Comment by ilovan** [comment]
not quite, I would say, @ajmacdonald , since The Moravian Lives schema has the same entity mappings as CWRC TEI lite )(and consequently should have NERVE assign the "proper" tags to identified named entities (i.e. persName, orgName, placeName, and NOT RS)

**Comment by ajmacdonald** [comment]
Ah, I had the wrong mappings for the Moravian schema. So it should be the CWRC Basic Schema mappings then.

**Comment by ilovan** [comment]
right - it should be identical to TEI Lite. 
@lucaju , I think you dealt with this - was it never pushed to dev: https://github.com/cwrc/CWRC-GitWriter/issues/65?

**Comment by lucaju** [comment]
It got pushed using `teiLite` as requested.
Does teiLite === CWRC Basic Schema mappings?
if not, perhaps it need to be changed?

**Comment by ajmacdonald** [comment]
It's confusing because the cwrc version's url includes tei lite: https://cwrc.ca/schemas/cwrc_tei_lite.rng
But it is different from the tei's tei lite: https://www.tei-c.org/release/xml/tei/custom/schema/relaxng/tei_lite.rng

**Comment by ilovan** [comment]
NOte that TEI Lite (the default, TEI consortium TEI Lite schema) uses RS only, but the CWRC TEI Lite schema is a CWRC TEI customization that does support nominal entity tags (persName, placeName, orgName)

**Comment by ajmacdonald** [comment]
Right. My point is just that it can be easy to confuse the two.

**Comment by ajmacdonald** [comment]
https://github.com/cwrc/CWRC-GitWriter/commit/9b20fc21b7df2ffe461a60195108e636be496dc3
